src/sidebar/services/test/root-thread-test.js

```diff
@@ -355,44 +343,4 @@ describe('rootThread', function () {
       );
     });
   });
-
-  context('when annotation events occur', function () {
-    const annot = annotationFixtures.defaultAnnotation();
-
-    it('creates a new annotation when BEFORE_ANNOTATION_CREATED event occurs', function () {
-      $rootScope.$broadcast(events.BEFORE_ANNOTATION_CREATED, annot);
-      assert.notCalled(fakeStore.removeAnnotations);
-      assert.calledWith(fakeAnnotationsService.create, sinon.match(annot));
-    });
-
-    describe('when a new annotation is created', function () {
-      let existingNewAnnot;
-      beforeEach(function () {
-        existingNewAnnot = { $tag: 'a-new-tag' };
-        fakeStore.state.annotations.annotations.push(existingNewAnnot);
-      });
-
-      it('does not remove annotations that have non-empty drafts', function () {
```

Comment:
This test and the one below was obsolete, but due to the way it works it still passed even though the code it used to test has been removed.